Gemfile

```diff
@@ -53,7 +53,11 @@ group(:development, :test) do
   gem "multi_json", "1.7.7", :require => false, :platforms => [:ruby, :jruby]
   gem "json-schema", "2.1.1", :require => false, :platforms => [:ruby, :jruby]
 
-  gem "rubocop", "~> 0.39.0", :platforms => [:ruby]
+  if RUBY_VERSION >= '2.0'
+    gem 'rubocop', '~> 0.49', :platforms => [:ruby]
```

Comment:
This makes us less restrictive in new versions of rubocop we automatically use. I'm generally for that, but I don't have any experience with Rubocop and how often its API changes (from the amount of work to go from 0.39 to 0.50.0 it seems like it breaks a bunch). Is this something folks who have worked on this project longer are on board with?

Reply:
I'm inclined to try and stay current if they break it again, we can pin to a known good version or fix it which ever is easier.